### PR TITLE
Create PR to review my work

### PR DESCRIPTION
**Pull request by sharma-shruti**
1) Add git_commands file.
2) Delete Powershell installation file from the server & push the changes on to Github repository.
3) Performed some git operations on the server and tested the git pull and push commands.

### CODE REVIEW

**Comment by abhi18av** [comment]
@sharma-shruti , please do use a more informative name

- Commit messages
- PR title

File: articles/git_commands.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+1.	Working area – The files in working area, are stored locally and are not handled by git. They can be discarded or can be moved to the staging area. Also called as untracked files.
+2.	Staging area - The staging area is how git knows what will change between the current commit and the next commit.
+3.	The repository - git knows about all the files in repository and contains all of your commits.
+
+## Headless OR Detached Head
+1.	Sometimes you need to checkout a specific commit (or tag) instead of a branch. 
+2.	git moves the HEAD pointer to that commit 
+3.	as soon as you checkout a different branch or commit, the value of HEAD will point to the new SHA 
+4.	There is no reference pointing to the commits you made in a detached state.
+```
```
Comment (abhi18av) [inline articles/git_commands.md:97]: You can use " ```sh "

File: articles/git_commands.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+root@bond:~/projects/My_Articles# git cat-file -p 9d8a4e7
+this is test file
+root@bond:~/projects/My_Articles# git cat-file -t 1c4cb79
+blob
+root@bond:~/projects/My_Articles# git cat-file -p 1c4cb79
+this is test file
+adding second line
+root@bond:~/projects/My_Articles#
+```
+*If you change any data about the commit, the commit will have
```
Comment (abhi18av) [inline articles/git_commands.md:70]: Good usage of italics here 👍

**Review by abhi18av — CHANGES_REQUESTED** [review]

File: articles/git_commands.md (hunk 1 above)
Comment (sharma-shruti) [reply articles/git_commands.md:97]: what does this mean?

Comment (abhi18av) [reply articles/git_commands.md:97]: You can specify the language inside the code-blocks in markdown.

Languages like `sh / ruby / powershell / ` etc

File: articles/Powershell_guide.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+PS /root> Get-Command -Noun service
+PS /root> get-service
+get-service: The term 'get-service' is not recognized as the name of a cmdlet, function, script file, or operable program.
+Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
+PS /root>
+
+PS /root> Get-Help Get-Service
+Get-Help: Get-Help could not find Get-Service in a help file in this session. To download updated help topics type: "Update-Help". To get help online, search for th
+e help topic in the TechNet library at https://go.microsoft.com/fwlink/?LinkID=107116.                                                                              
+PS /root>
```
Comment (abhi18av) [inline articles/Powershell_guide.md]: @sharma-shruti ,

This needs to be in a triple single quotes as a script block.


This comment is repeated on almost every PR 😞

**Review by abhi18av — CHANGES_REQUESTED** [review]

File: articles/Powershell_guide.md (hunk 3 above)
Comment (sharma-shruti) [reply articles/Powershell_guide.md]: I didn't mean to include this file in the PR, as I still have some unfinished work. All I was doing is saving my work in my branch origin/develop but it looks like the file got pushed on to origin/master. How can we exclude the file, from this PR?

File: file1.txt [hunk 4]
```diff
@@ -0,0 +1,2 @@
+this is test file
+adding second line
```
Comment (abhi18av) [inline file1.txt:2]: Why do we need this file?

File: hello.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+Hello, World!
```
Comment (abhi18av) [inline hello.txt:1]: Or this one?

File: file1.txt (hunk 4 above)
Comment (sharma-shruti) [reply file1.txt:2]: I added some test files while doing my labs from the server.

File: articles/Powershell_guide.md (hunk 3 above)
Comment (sharma-shruti) [reply articles/Powershell_guide.md]: This file is no longer in the Github repository. I have reset the HEAD to the previous commit using ```git reset HEAD~1 ``` command

**Pull request merged**